test/test_example.py

```diff
@@ -40,7 +40,7 @@ def test_csv_to_aligned_hierarchies_file_saved(self):
         thresh = 0.01
 
         csv_to_aligned_hierarchies(file_in, file_out, num_fv_per_shingle,
-                                   thresh)
+                                   thresh, True)
```

Comment:
why is vis=True here but not in line 28?

Reply:
I just want to test if vis=False, the function can work normally. Do you think I should change it to True for consistency?

Reply:
Oh that's fine then, I just didn't know the intent.